i18n/en.json

```diff
@@ -1895,6 +1896,9 @@
   "delete_post.post": "Post",
   "delete_post.question": "Are you sure you want to delete this {term}?",
   "delete_post.warning": "This post has {count, number} {count, plural, one {comment} other {comments}} on it.",
+  "demote_to_user_modal.demote": "Demote",
+  "demote_to_user_modal.desc": "This action demotes the user {username} to be a guest. The wont be able to contact with users outside of his channels or join to public channels by themselves. Are you sure you want to demote user {username} to guest?",
```

Comment:
Recommend:  "This action demotes the user {username} to a guest. This action will restrict the user's ability to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to demote user {username} to guest?"

Reply:
Done :)